### PR TITLE
Razor Update

### PR DESCRIPTION
**Pull request by remuluson2**
## About The Pull Request
Added electric razor to lathes in medical section and added it to the library vendor since hair dye is already there.

## Why It's Good For The Game
If you want to give someone a haircut but someone takes the only razor on the station, you need to buy a surgical crate to get another one. This PR gives an easy way to make more razors for all the hairdressers out there.

## Changelog

:cl:
add: Electric razors can now be printed at any lathe. Its in medical section.
add: Electric razor can be bought from the library vendor.
/:cl:

### CODE REVIEW

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -1221,3 +1221,10 @@
 	materials = list(/datum/material/iron = 100, /datum/material/glass = 50)
 	build_path = /obj/item/assembly/control
 	category = list("initial","Misc")
+/datum/design/razor
```
Comment (Mothblocks) [inline code/modules/research/designs/autolathe_designs.dm:1224]:
```suggestion

/datum/design/razor
```

consistent with the rest of the file

File: code/modules/research/designs/autolathe_designs.dm [hunk 2]
```diff
@@ -1221,3 +1221,10 @@
...
+	name = "Electric Razor"
+	id = "razor"
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+	materials = list(/datum/material/iron = 75)
+	build_path = /obj/item/razor
+	category = list("initial","Medical")
```
Comment (Mothblocks) [inline code/modules/research/designs/autolathe_designs.dm:1230]: is it really medical though

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/research/designs/autolathe_designs.dm (hunk 2 above)
Comment (remuluson2) [reply code/modules/research/designs/autolathe_designs.dm:1231]: Acording to cargo crates yes, but I guess I can move it to Misc, my problem is that misc is just really really packed and I dont wanna pack it more

**Comment by remuluson2** [comment]
uhhh. did it merge the gap between the designs?

File: code/modules/research/designs/autolathe_designs.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/research/designs/autolathe_designs.dm:1231]: you didnt put this in techwebs its not going to be printable in medlathe, also you didnt set a departmental flag

Comment (remuluson2) [reply code/modules/research/designs/autolathe_designs.dm:1231]: oh right. I just copied how other tools were put in. I intend it to be printable everywhere in medical section.

Comment (remuluson2) [reply code/modules/research/designs/autolathe_designs.dm:1231]: not just in medical. Altho I can move it to autolathe only, There is only one autolathe tho so I am on the fence.

Comment (remuluson2) [reply code/modules/research/designs/autolathe_designs.dm:1231]: alright. i am moving it to autolathe and service lathe. it stays in medical section tho cuz yes

**Comment by LemonInTheDark** [comment]
It occurs to me there's all these items in cargo that none buys because dealing with cargo is a headache, so we put them on station which kinda makes cargo progressively more useless.

Wonder if anything can be done about that pattern, something to make cargo more interested in outside orders. Nothing against this pr, seems ok, it's just something I've noticed

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -157,6 +157,7 @@
 		"wirebrush",
 		"wirecutters",
 		"wrench",
+		"razor",
```
Comment (Mothblocks) [inline code/modules/research/techweb/all_nodes.dm:160]: alphabetical order

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by necromanceranne** [comment]
> It occurs to me there's all these items in cargo that none buys because dealing with cargo is a headache, so we put them on station which kinda makes cargo progressively more useless.
> 
> Wonder if anything can be done about that pattern, something to make cargo more interested in outside orders. Nothing against this pr, seems ok, it's just something I've noticed

Ever lost a package with a mailing company? Imagine that, but the package you shipped doesn't even have your name or address on it and you need to go into the mailing office across town to verify the package is yours, the van that delivered the goods to that office is liable to drive off and lose your goods because it's operated remotely by monkeys playing with a controller, and the people handling the package aren't just stealing your goods, they're also going to shoot you when you complain.

That's cargo. It's basically set up to fail at it's job on multiple levels.

**Pull request merged**